Formula/ortp.rb

```diff
@@ -20,8 +20,8 @@ class Ortp < Formula
   # bctoolbox appears to follow ortp's version. This can be verified at the GitHub mirror:
   # https://github.com/BelledonneCommunications/bctoolbox
   resource "bctoolbox" do
-    url "https://gitlab.linphone.org/BC/public/bctoolbox/-/archive/4.4.29/bctoolbox-4.4.29.tar.bz2"
-    sha256 "587940afa2fbe2ee701d56c3f09acbca36c71fc613a7a4b41770a2f6102bd190"
+    url "https://gitlab.linphone.org/BC/public/bctoolbox/-/archive/4.4.31/bctoolbox-4.4.30.tar.bz2"
```

Comment:
Shouldn't it be ?
```suggestion
    url "https://gitlab.linphone.org/BC/public/bctoolbox/-/archive/4.4.31/bctoolbox-4.4.31.tar.bz2"
```

Reply:
Whoops. Though, I am surprised that is a working URL

Reply:
It actually just needs the `x.bz2` extension to work 😅